libs/perun/components/src/lib/attributes-list/attribute-value/attribute-value-integer/attribute-value-integer.component.ts

```diff
@@ -19,7 +23,20 @@ export class AttributeValueIntegerComponent {
 
   @Output() sendEventToParent = new EventEmitter();
 
+  ngOnInit() {
+    if(!this.readonly){
+      this.readonly = new IsVirtualAttributePipe().transform(this.attribute);
```

Comment:
It might be worth extracting the code from the pipe to the perun-utils.ts file, so you don't need to create a pipe.